samples/HealthChecksSample/Program.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace HealthChecksSample
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            BuildWebHost(args).Run();
+        }
+
+        public static IWebHost BuildWebHost(string[] args) =>
+            new WebHostBuilder()
```

Comment:
Reason to not use `CreateDefaultBuilder`?

Reply:
Because I didn't. Can change.

Reply:
You can't, none of our repos can use CreateDefaultBuilder without creating a circular dependency.